docs/docsite/rst/plugins/connection.rst

```diff
@@ -9,7 +9,7 @@ Connection plugins
 
 Connection plugins allow Ansible to connect to the target hosts so it can execute tasks on them. Ansible ships with many connection plugins, but only one can be used per host at a time.
 
-By default, Ansible ships with several connection plugins. The most commonly used are the :ref:`paramiko SSH<paramiko_connection>`, native ssh (just called :ref:`ssh<ssh_connection>`), and :ref:`local<local_connection>` connection types.  All of these can be used in playbooks and with :command:`/usr/bin/ansible` to decide how you want to talk to remote machines. If necessary, you can :ref:`create custom connection plugins <developing_connection_plugins>`.
+By default, Ansible ships with several connection plugins. The most commonly used are the :ref:`paramiko SSH<paramiko_connection>`, native ssh (just called :ref:`ssh<ssh_connection>`), and :ref:`local<local_connection>` connection types.  All of these can be used in playbooks and with :command:`/usr/bin/ansible` to decide how you want to talk to remote machines. If necessary, you can :ref:`create custom connection plugins <developing_connection_plugins>`. To establish the connection, you can use an ``ansible-connection`` script referenced in :ref:`ANSIBLE_CONNECTION_PATH<ANSIBLE_CONNECTION_PATH>`. To change the connection plugin used for tasks, you can use the ``connection`` keyword.  
```

Comment:
I'm not so sure about the description of `ANSIBLE_CONNECTION_PATH` here. The `ansible-connection` script is installed by default in the same directory as the `ansible` executable.

I think the environment variable is useful to point to a different location for custom files for doing things like establishing connections and managing remote execution. So it might be better to say something along the lines of:

"You can use the `ANSIBLE_CONNECTION_PATH` environment variable to specify a custom directory for the ``ansible-connection`` script along with other files related to connections during remote execution."